uhaul/components/order_models/order.py

```diff
@@ -14,4 +14,4 @@ def set_data(self, data: dict):
             if not hasattr(self, k):
                 raise ValueError(f"No such attribute: {k}")
 
-            setattr(self, k, v)
\ No newline at end of file
+            setattr(self, k, v)
```

Comment:
The removed and added code looks identical, is it because of some whitespace?

Reply:
Yes, that's correct, nothing important here